app/containers/TuneContainer/tests/reducer.test.js

```diff
@@ -59,4 +59,43 @@ describe('TuneContainer reducer tests', () => {
       })
     ).toEqual(expectedResult);
   });
+
+  it('should return the state when an action of type FETCH_TRACK_DETAILS is dispatched', () => {
```

Comment:
let's say `should  update` initial state with `songId` when action.... is dispatched.